Dockerfile

```diff
@@ -1,9 +1,9 @@
 # Runs master branch from github of lando/lando-messaging
 # Requires a lando config file mounted at /etc/lando_config.yml.
 FROM python:2.7.13
-RUN pip install --upgrade git+git://github.com/Duke-GCB/lando-messaging.git
 ADD . /src
 WORKDIR /src
+RUN pip install --upgrade git+git://github.com/Duke-GCB/lando-messaging.git
```

Comment:
Still don't need the `--upgrade`, and it would be good to add a comment here on why the dependency is added after `src` (e.g. to make sure old version is not cached after local src changes)

And this won't rebuild if we update lando-messaging without updating lando, but it's close to what we want.